acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicenceAPIPage.java

```diff
@@ -59,13 +74,123 @@ public void postRequestToSessionEndpoint() throws IOException, InterruptedExcept
         Map<String, String> deserialisedResponse =
                 objectMapper.readValue(sessionResponse, new TypeReference<>() {});
         SESSION_ID = deserialisedResponse.get("session_id");
+        STATE = deserialisedResponse.get("state");
     }
 
-    public void getSessionId() {
+    public void getSessionIdForDL() {
         LOGGER.info("SESSION_ID = " + SESSION_ID);
         assertTrue(StringUtils.isNotBlank(SESSION_ID));
     }
 
+    public void postRequestToDrivingLicenceEndpoint(String dlJsonRequestBody)
+            throws IOException, InterruptedException {
+        String privateApiGatewayUrl = configurationService.getPrivateAPIEndpoint();
+        JsonNode dlJsonNode =
+                objectMapper.readTree(
+                        new File("src/test/resources/Data/" + dlJsonRequestBody + ".json"));
+        String dlInputJsonString = dlJsonNode.toString();
+        HttpRequest request =
+                HttpRequest.newBuilder()
+                        .uri(URI.create(privateApiGatewayUrl + "/check-driving-licence"))
+                        .setHeader("Accept", "application/json")
+                        .setHeader("Content-Type", "application/json")
+                        .setHeader("session_id", SESSION_ID)
+                        .POST(HttpRequest.BodyPublishers.ofString(dlInputJsonString))
+                        .build();
+        LOGGER.info("drivingLicenceRequestBody = " + dlInputJsonString);
+        String drivingLicenceCheckResponse = sendHttpRequest(request).body();
+        LOGGER.info("drivingLicenceCheckResponse = " + drivingLicenceCheckResponse);
+        Map<String, String> deserialisedResponse =
```

Comment:
This could be a DocumentCheckResponse rather than a generic map. Makes it easier to read an reason about